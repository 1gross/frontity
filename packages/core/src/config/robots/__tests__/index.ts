import getRobots from "../";

test("Babel returns for development", () => {
  expect(getRobots({ mode: "development" })).toMatchSnapshot();
});

test("Babel returns for production", () => {
  expect(getRobots({ mode: "production" })).toMatchSnapshot();
});
