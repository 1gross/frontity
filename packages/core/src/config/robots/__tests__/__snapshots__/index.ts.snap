// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Babel returns for development 1`] = `
"User-agent: *
Disallow:"
`;

exports[`Babel returns for production 1`] = `
"User-agent: *
Disallow:"
`;
